Dockerfile

```diff
@@ -1,5 +1,13 @@
 FROM node:16-alpine AS build
 
+LABEL \
+    org.opencontainers.image.title="http-https-echo" \
+    org.opencontainers.image.description="Docker image that echoes request data as JSON; listens on HTTP/S, with various extra features, useful for debugging." \
+    org.opencontainers.image.url="https://github.com/mendhak/docker-http-https-echo" \
+    org.opencontainers.image.documentation="https://github.com/mendhak/docker-http-https-echo/blob/master/README.md" \
+    org.opencontainers.image.source="https://github.com/mendhak/docker-http-https-echo" \
+    org.opencontainers.image.licenses="MIT"
```

Comment:
For these labels to be in the final build, they need to be below `FROM node:16-alpine AS final` further down this file.

Reply:
hehe missed that.  OK new tag. 

docker pull mendhak/http-https-echo:labels
docker inspect lscr.io/linuxserver/transmission --format '{{ .Config.Labels }}'